internal/master/service.go

```diff
@@ -58,6 +58,17 @@ func (ss *standaloneService) Put(ctx context.Context, putReq *serverpb.PutReques
 	return &serverpb.PutResponse{Status: newEmptyStatus()}, nil
 }
 
+func (ss *standaloneService) Delete(ctx context.Context, delReq *serverpb.DeleteRequest) (*serverpb.DeleteResponse, error) {
```

Comment:
Similar functionality is needed inside the `distributedService` impl. in the same file. Are you planning to take this up in a later PR ?

Reply:
Thanks for finding this one, I missed out on this one. Have made the required changes.